spec/dummy/app/models/card.rb

```diff
@@ -0,0 +1,35 @@
+require 'time'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.